Sources/ChatGPTSwift/ChatGPTAPI.swift

```diff
@@ -57,7 +57,9 @@ public class ChatGPTAPI: @unchecked Sendable {
     private func generateMessages(from text: String, systemText: String) -> [Message] {
         var messages = [systemMessage(content: systemText)] + historyList + [Message(role: "user", content: text)]
         if gptEncoder.encode(text: messages.content).count > 4096  {
+            if !historyList.isEmpty() {
```

Comment:
Why `isEmpty` is a function?

Reply:
Thanks! Working on this now. I'm very new to Swift and made this edit in a github codespace. This solution doesn't work anyway. Something more complex is required.